app/Main.hs

```diff
@@ -19,11 +20,24 @@ type Deck = [Card]
 
 -- | `loadDeck` is a computation which tries to load a deck from disk.
 loadDeck :: IO (Maybe Deck)
-loadDeck = undefined
+loadDeck = do
+    putStr "Enter Filename (without extension): "
+    fn <- getLine
+    deck <- decodeFileStrict $ fn ++ ".json"
```

Comment:
`<.>` from `System.FilePath` (`filepath` package) may be nice to use for this

Reply:
Done with https://github.com/fpclass/perpetual-haskelling-initiative/pull/7/commits/413196fb0ce392600d43e95be617bd04728004ed